.circleci/config.yml

```diff
@@ -274,7 +265,9 @@ jobs:
             git tag v${GUS_BUILD}
             git push origin v${GUS_BUILD}
             git push origin master
-      - run: *gus-close-change-mgmt
+      - sfchangecase/update:
```

Comment:
The default status is to close it. See the [internal repo](https://github.com/forcedotcom/change-case-management-orb/blob/master/src/commands/update.yml#L13)